### ISSUE TITLE
[Bug]; The App cannot grade the test

### ISSUE THREAD

**Issue by plantindesk** [quenti-io/quenti#337]
## Problem
The app doesn't grade the test. Also, this only happens when I select 'Write' option too and we get these errors everytime

![image](https://github.com/user-attachments/assets/7b7d6660-ee58-4c87-9e23-14b32c60c6ef)

![image](https://github.com/user-attachments/assets/8e274227-7e63-43dd-a96e-a1cb515810e9)

**Comment by pihedron** [comment]
Yes, I have also noticed that the app doesn't grade the test for questions where you need to write the definition. I think there is a bug with the cortex *AI* feature. If you want to manually check your answers, you can expand the `input` field on the objects and view the correct answer via console. This is still a critical bug though.

I think it would be better for users to be given the option to mark write questions themselves if cortex doesn't work. For now, I recommend using learn mode or writing terms down on paper. It's usually better to have some physical element to revision.
